### DISCUSSION TITLE
Pixel clustering with Pixie - error

### DISCUSSION THREAD

**Discussion by Nitya20001** [angelolab/ark-analysis#1089] [Announcements]
![MicrosoftTeams-image (8)](https://github.com/angelolab/ark-analysis/assets/130975488/62f65621-4d83-48db-8832-d68738bda390)

**Describe the bug**
Not sure if this is a bug: This is my first time running the pixie notebook. I added my tiff files for each channel, the deep cell segmentation output, etc in the data folder that was created when I downloaded the ark analysis package.

However, when I run the function below to create the pixel SOM weights:

pixel_pysom = pixel_som_clustering.train_pixel_som(
    fovs,
    channels,
    base_dir,
    subset_dir=pixel_subset_dir,
    norm_vals_name=norm_vals_name,
    som_weights_name=pixel_som_weights_name,
    num_passes=1,
    seed=42
)

I get this error:

The file/path, channel_norm_post_rownorm.feather, could not be found..

Please help!

**Comment by alex-l-kong** [comment]
@Nitya20001 we changed a lot of the naming conventions for pixel clustering, so you may have an out-of-date codebase. Can you first update your repo by running at the root of the `ark-analysis` folder:

```
git stash
git pull
```

Note that this will update all Jupyter notebooks in the `templates` folder as well. Any copies that you've been working with may be out-of-date or overwritten. So make a new copy of these base notebooks prior to running.

Then update the internal installation by running:

```
pip install -e . --user
```

Make sure you've activated your `ark_env` first.

Afterwards, try re-running from scratch and see if that resolves the issue.

**Comment by Nitya20001** [comment]
Hi I think the issue may be something else. I am actually trying to run the pipeline on CellDIVE images (a fluorescence based multiplexed imaging technique) and I have been able to run DeepCell and create a segmentation mask but I only have one 'FOV'. My files are arranges as described on the pixie githib but I don't have different FOV folders for each channel. Do you think adding that in would work?

**Reply by alex-l-kong** [reply]
The pipeline does require a folder directory structure with FOVs as the top-level dirs, so yes please try that out. We had to do that on one cohort at Angelo Lab.
